pages/services/chat.html

```diff
@@ -50,7 +50,7 @@ <h4>Is #general:privacytools.io or #privacy:privacytools.io chat end-to-end encr
 <p><strong>No.</strong> Messages sent to the #general chat are encrypted from with client-to-server encryption, i.e. HTTPS. This means that your messages can't be viewed by an attacker on your network, or your ISP, etc. <strong>However</strong>, because this is a public chat room anybody can view your messages if they are a member, and newcomers will be able to see all message history. Do not say anything in rooms that don't use End-to-End (E2E) encryption that you wouldn't want tied to you personally.</p>
 
 <h4>Isn't Matrix behind Cloudflare?</h4>
-<p>Matrix.org is, chat.privacytools.io isn't. If you use our homeserver and our Element install, you should be good.</p>
+<p>The default matrix.org homeserver is behind Cloudflare. The PrivacyTools homeserver (<a href="https://element.privacytools.io/">element.privacytools.io</a> is not.
```

Comment:
```suggestion
<p>The default matrix.org homeserver is behind Cloudflare. The PrivacyTools homeserver (<a href="https://chat.privacytools.io/">chat.privacytools.io</a> is not.
```

Element still seems to be the webclient while chat. says to be the homeserver.

Reply:
Ah yes. good point.